Makefile

```diff
@@ -37,4 +37,8 @@ $(foreach v,${HELPERS},$(eval $(call add-scripts,helpers,$v)))
 
 install: $(all-install)
 
+install_auto: init/bootrr-auto.sh
+	echo "INSTALL $?"
+	install -D -m 755 $? $(DESTDIR)/etc/profile.d/bootrr-auto.sh
```

Comment:
Shouldn't this be $(DESTDIR)$(prefix)/etc... ?

Reply:
@ndechesne pointed out that this should not be /usr, so we're good on this front. But we should get the bootrr-auto.sh into the cpio archive as well.